ansible/roles/providers/tasks/vmware.yml

```diff
@@ -25,6 +25,14 @@
       - cloud-initramfs-dyn-netconf
   when: ansible_os_family == "Debian"
 
+- name: Ensure cloud-init version > 23.1.3 is installed
+  apt:
+    name: "cloud-init=24.1.3-0ubuntu1~20.04.5"
```

Comment:
This pins the version, right? Is that what we want to do here or allow > 23.1.3?

Reply:
i added this commit to sort of force the situation that the reported bugs came in -

without this commit i was not able to get a new version of cloud init for some reason. 

We can keep it here, if we want to force a newer version